api/CustomElementRegistry.json

```diff
@@ -0,0 +1,65 @@
+{
+  "api": {
+    "CustomElementRegistry": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/CustomElementRegistry",
+        "description": "The CustomElementRegistry interface provides methods for registering custom elements and querying registered elements.",
+        "support": {
+          "chrome": {
+            "version_added": "33",
+            "notes": "Chrome 36+/Opera 20+ implemented a previous version of Custom Elements (v0) that used `.registerElement()`"
```

Comment:
"chrome_android" has an additional restriction note, as well as safari. Doesn't it apply to "chrome" too? (And Opera): if so, they have to be added to each fo them.

Reply:
@teoli2003 how do we do multiple notes...an array? Perhaps i need to RTFM more hehe. ;-)

Fixed in: https://github.com/mdn/browser-compat-data/pull/623/commits/f8186a5b4ef483fce0193cae09983ad646d2a50a